ibm/service/power/data_source_ibm_pi_volume_onboarding.go

```diff
@@ -0,0 +1,133 @@
+// Copyright IBM Corp. 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/power-go-client/helpers"
+	"github.com/IBM-Cloud/power-go-client/power/models"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+)
+
+func DataSourceIBMPIVolumeOnboarding() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIBMPIVolumeOnboardingReads,
+		Schema: map[string]*schema.Schema{
+			PIVolumeOnboardingID: {
+				Type:         schema.TypeString,
+				Required:     true,
+				Description:  "Volume onboarding ID",
+				ValidateFunc: validation.NoZeroValues,
+			},
+			helpers.PICloudInstanceId: {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+
+			// Computed Attributes
+			"create_time": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Indicates the create time of volume onboarding operation",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Description of the volume onboarding operation",
+			},
+			"id": {
```

Comment:
This argument declaration is not required since Terraform already provide id for each res/ds

Reply:
done